src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -1838,9 +1884,11 @@ module Queries = struct
   let coda_constants =
     field "codaConstants" ~doc:"Coda constants"
```

Comment:
Do you think it could make sense to call the new field "consensusConstants"? I don't know exactly what all these values are, or if they're all related to consensus, but it might be more descriptive than codaConstants